scripts/cangen/main.py

```diff
@@ -68,6 +69,9 @@ def parse():
     bus_list = config["canGen"]["busses"]
     output_path = config["canGen"].get("outputPath", DEFAULT_OUTPUT_DIR)
 
+    # Deletes output path folder and files within, before creating new ones
+    shutil.rmtree(output_path)
```

Comment:
sorry I should have been more clear, you should still wrap this inside `if os.path.exists(...)`